app/controllers/projects_controller.rb

```diff
@@ -72,11 +72,15 @@ def index
                         [p, Sample.where(project: p).count]
                       end
                     end
+        # TODO: (gdingle): this seems inefficient... 5 extra queries * projects.length
```

Comment:
@tfrcarvalho told me to leave the data queries alone until his refactor is in.

Reply:
Thanks for flagging @gregdingle ! We may have more such patterns in the codebase and it's good to get rid of them. For this particular place, I already have a PR open, still to be tested: https://github.com/chanzuckerberg/idseq-web/pull/1976
Feel free to just incorporate anything you find useful into your PR because I may not have time to get to the testing immediately.

@tfrcarvalho Is there a branch we can look at for the refactor you mentioned? Or would it actually help alleviate some of your workload if one of us fixed these inefficiencies right now? I know you have taken on a ton of work for data discovery, which is amazing, it's such a huge project. In general I dislike leaving critical fixes such as this one for later! So I would strongly suggest either I merge my PR in or Greg incorporates the changes into his. People will copy and paste this code pattern and it will proliferate like bunny rabbits ;)

Reply:
It's quite worrying to me that just in this PR, we're already adding 3 more inefficient lines just because we want to leave the refactor for later

Reply:
One option: since @gregdingle  is splitting the bug fix from this PR... lets leave it like this until my PR is in. Then I can submit a change to this one correcting the endpoints and connecting the data.
Second option: submit the frontend code only and I can connect it in a separate PR. 

I am already working on providing the data for the old code and will work on this soon.
Lets not copy more refactors from each other... that just confusing and hard to merge later.